src/components/ApplicationGroup/Details/EnvironmentOverview/service.ts

```diff
@@ -0,0 +1,9 @@
+import { post } from "@devtron-labs/devtron-fe-common-lib"
+
+export const hibernate = async (appIds:number[],envId:number,envName:string) => {
+    return post(`batch/v1beta1/hibernate`, { appIdIncludes: appIds, envId, envName })
+}
+
+export const unhibernate = async (appIds:number[],envId:number,envName:string) => {
+    return post(`batch/v1beta1/unhibernate`, { appIdIncludes: appIds, envId, envName })
+}
```

Comment:
- These two functions can be combined into a single function and use a type: 'hibernate' | 'unhibernate' to control the url
- Also use direct arrow function returns